src/ReinforcementLearningZoo/src/algorithms/policy_gradient/vmpo.jl

```diff
@@ -2,6 +2,7 @@ export VMPOPolicy, VMPOTrajectory
 
 using LinearAlgebra: ⋅  # dot product
 using ChainRulesCore: ignore_derivatives
+import ReinforcementLearningCore.normlogpdf
```

Comment:
Should we make this consistent with the first example and do:
```suggestion
import ReinforcementLearningCore: normlogpdf
```